File: tests/benchmarks/cases/matrix_add_2d/dotnet/benchmark_case.cs
namespace BenchmarkRunner;

internal static class MatrixAdd2dCase
{
    public const string Name = "matrix_add_2d";
    public const string PassBanner = "BENCH_MATRIX_ADD_2D_PASS";

    public static long Run(int scale) => BenchmarkSupport.MatrixAdd2d(scale);
}

File: tests/benchmarks/cases/matrix_add_2d/java/benchmark_case.java
final class MatrixAdd2dCase {
    static final String NAME = "matrix_add_2d";
    static final String PASS_BANNER = "BENCH_MATRIX_ADD_2D_PASS";

    private MatrixAdd2dCase() {}

    static long run(int scale) {
        return BenchmarkSupport.matrixAdd2d(scale);
    }
}
